firebase-sessions/src/main/kotlin/com/google/firebase/sessions/settings/SessionsSettings.kt

```diff
@@ -24,26 +25,52 @@ import kotlin.time.Duration.Companion.minutes
  *
  * @hide
  */
-internal class SessionsSettings {
+internal class SessionsSettings(val context: Context) {
+
+  var localOverrideSettings = LocalOverrideSettings(context)
+
+  // Order of preference for all the configs below:
+  // 1. Honor local overrides
+  // 2. If no local overrides, use remote config
+  // 3. If no remote config, fall back to SDK defaults.
+
   // Setting to qualify if sessions service is enabled.
   val sessionsEnabled: Boolean
     get() {
+      if (localOverrideSettings.sessionEnabled != null) {
+        return localOverrideSettings.sessionEnabled!!
```

Comment:
Instead of this pattern and using `!!`, Kotlin has nice syntax for dealing with null like:

Elvis if it's just 1 or the other:
`get() =  localOverrideSettings.sessionEnabled :? true`

There is also this for when we have multiple cases:
```
localOverrideSettings.sessionEnabled?.let { return it }

someOtherCases?.let { return it }

return true
```